lib/open3.rb

```diff
@@ -1061,57 +1061,37 @@ def pipeline_start(*cmds, &block)
   end
   module_function :pipeline_start
 
-  # Open3.pipeline starts a list of commands as a pipeline.
-  # It waits for the completion of the commands.
-  # No pipes are created for stdin of the first command and
-  # stdout of the last command.
+  # :call-seq:
+  #   Open3.pipeline([env, ] *cmds, options = {}) -> [stdin, stdout, wait_thread] -> statuses
```

Comment:
Is this part a typo?

```suggestion
  #   Open3.pipeline([env, ] *cmds, options = {}) -> [stdin, stdout, wait_thread]
```